### DISCUSSION TITLE
Potential on FlxVersion

### DISCUSSION THREAD

**Discussion by TheHENOOB** [HaxeFlixel/flixel#2617] [General]
In my opinion, flixel.system.FlxVersion can be an potential helper class to check the versions of an game, not only on the engine, giving standard data from the name of the game or library (that the game is using), the version of it (major, minor and patch which can be optional) etc.

From now, FlxVersion is only limited to use the HaxeFlixel label.

https://github.com/HaxeFlixel/flixel/blob/dev/flixel/system/FlxVersion.hx

**Comment by RichardBray** [comment]
That's an interesting point. 

For my games, I tend to use Lix and NPM and use the [package.json](https://github.com/RichardBray/yeti-project/blob/main/package.json) file for [versioning the game](https://github.com/RichardBray/yeti-project/blob/main/src/states/GameState.hx#L35). 

I can use libraries like [standard-version](https://github.com/conventional-changelog/standard-version) to change the game version and that will be displayed wherever I want in the game. But I know that technique isn't for everyone. Happy to make a video on it or something if you're interested.

**Comment by TheScratchTeen** [comment]
`FlxVersion` is made for flixel library, because when using `new FlxVersion(1, 2, 3).toString()`, it returns `HaxeFlixel 1.2.3`!
Try to make your custom class. I wanted to make my own one too.

**Comment by Geokureli** [comment]
> In my opinion, flixel.system.FlxVersion can be an potential helper class to check the versions of an game, not only on the engine, giving standard data from the name of the game or library (that the game is using), the version of it (major, minor and patch which can be optional) etc.
> 
> From now, FlxVersion is only limited to use the HaxeFlixel label.

This should help
https://gist.github.com/Geokureli/9d0ac4598aa595b79f43919850601701
